e2e/src/fixtures/FixtureConfigs.ts

```diff
@@ -56,4 +56,77 @@ export const expect = baseExpect.extend({
       actual: matcherResult?.actual,
     };
   },
+
+  async toHaveSplitText(locator: Locator, expected: string) {
+    const assertionName = 'toHaveSplitText';
+    let pass: boolean;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let matcherResult: any;
+    const text: string | null = await locator.textContent();
+
+    try {
+      if (text === null) {
+        throw new Error('Element has no content');
+      }
+
+      if (!text.includes(expected)) {
+        throw {
+          matcherResult: {
+            actual: text,
+            expected,
+          }
+        };
+      }
```

Comment:
The try / catch block is piled up. Also, the if (!text.includes(expected)) check doesn't give a clear reason for the error.